src/Languages/pt-BR.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Language Name="Brazilian Portuguese">
+<Language Name="Português (Brasil)">

```

Comment:
I used the native names provided by Windows:
```
PS> [System.Globalization.CultureInfo]::GetCultureInfo('pt-BR').NativeName
Português (Brasil)
PS> [System.Globalization.CultureInfo]::GetCultureInfo('pt-PT').NativeName
português (Portugal)
```
You might get better data from [CLDR.](http://cldr.unicode.org/)
Download core.zip from [CLDR Releases/Downloads/latest-data.](http://unicode.org/Public/cldr/latest)

I was surprised to see
```
PS> [System.Globalization.CultureInfo]::GetCultureInfo('nl-NL').NativeName
Nederlands (Nederland)
```
 because we (Dutch) definitely do *not* capitalise language names when used in a sentence.